wails-frontend/src/pages/PasswordPromptHandler/Default.tsx

```diff
@@ -82,7 +86,7 @@ export function Default() {
             defaultValue=""
             name="password"
             placeholder="Password"
-            error={error?.password}
+            error={errorMessage ?? error?.password}
```

Comment:
does the UI render pretty?  I would expect a separated `<p>` bellow.

Reply:
ui does propoerly render errorMessage if not null or fallsback to error.password

Reply:
<img width="675" alt="Screenshot 2023-08-25 at 14 29 01" src="https://github.com/massalabs/station-massa-wallet/assets/90157528/47f0ea78-a2e7-42ec-b1fa-d3a9df3c2b81">

test method : ``` if (!Success) {
      if (CodeMessage === ErrorCode.WrongPassword) {
        setErrorMessage('Failed to process transaction');
        return;
      } else {
        setError({ password: Intl.t(`errors.${CodeMessage}`) });
        return;
      }
    }```

Reply:
but i guess most optimal would be to just reuse the errorObject we already have, that way we can remove the errorMessage useState, wdyt @Thykof ?

Reply:
we should not mix errors related to a field of the form and error related to the submit action

Reply:
see for example here
https://github.com/massalabs/station-massa-wallet/blob/6c317bd32a8608bf55ce9d772acf4092931b0e83/wails-frontend/src/pages/backupMethods.tsx#L49

we have an independent local state and html tag

Reply:
true but in your example, the error does not live in a component; here we have the input component which wraps automatically the value of ```error``` in a div tag -> https://github.com/massalabs/ui-kit/blob/main/src/components/Input/Input.tsx

Reply:
understood ! will keep error render separate !

Reply:
<img width="525" alt="Screenshot 2023-08-25 at 15 42 20" src="https://github.com/massalabs/station-massa-wallet/assets/90157528/41bcd50d-a21a-4a60-a7d4-280b4e2821c9">